recipes/fast-double-parser/all/conanfile.py

```diff
@@ -0,0 +1,36 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.43.0"
+
+
+class FastDoubleParserConan(ConanFile):
+    name = "fast-double-parser"
+    description = "Fast function to parse strings into double (binary64) floating-point values, enforces the RFC 7159 (JSON standard) grammar: 4x faster than strtod"
+    topics = ("numerical", "header-only")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/lemire/fast_double_parser"
+    license = "Apache-2.0"
+
+    settings = "os", "compiler"
```

Comment:
```suggestion
    settings = "os", "compiler", "build_type", "arch"
```

Keep full settings by default. You can manage the package ID

Reply:
Thank you for reviewing! I fixed it.